plone/app/dexterity/permissions.py

```diff
@@ -82,14 +82,24 @@ def validate(self, field_name, vocabulary_name=None):
             raise AttributeError('No such field: {0}'.format(field_name))
 
 
-class DXAddViewFieldPermissionChecker(DXFieldPermissionChecker):
+class GenericFormFieldPermissionChecker(DXFieldPermissionChecker):
```

Comment:
renaming this is fine, but please place a BBB import with deprecation warning in order to not break any addons imports.